src/components/header/EmptyThreadIndicator.css

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.headerEmptyThreadIndicator {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```

Comment:
This is a nit, and I'm not sure this has an impact, but I usually do `top: 0; left: 0; right: 0; bottom: 0` for this.